test/e2e/minscale_readiness_test.go

```diff
@@ -50,24 +50,31 @@ func TestMinScale(t *testing.T) {
 		Image:  "helloworld",
 	}
 
+	test.CleanupOnInterrupt(func() { test.TearDown(clients, names) })
+	defer test.TearDown(clients, names)
+
 	t.Log("Creating configuration")
 	if _, err := v1a1test.CreateConfiguration(t, clients, names, withMinScale(minScale)); err != nil {
 		t.Fatalf("Failed to create Configuration: %v", err)
 	}
 
-	test.CleanupOnInterrupt(func() { test.TearDown(clients, names) })
-	defer test.TearDown(clients, names)
-
 	revName := latestRevisionName(t, clients, names.Config)
 	deploymentName := revName + "-deployment"
 
+	sks, err := clients.NetworkingClient.ServerlessServices.Get(revName, metav1.GetOptions{})
+	if err != nil {
+		t.Fatalf("Error retrieving sks %q: %w", revName, err)
```

Comment:
`%w` works only with `fmt.Errorf`.
```suggestion
		t.Fatalf("Error retrieving sks %q: %v", revName, err)
```